packages/paste-website/src/component-examples/CornerOrnamentExamples.ts

```diff
@@ -0,0 +1,283 @@
+export const featuredExample = `
+const FeaturedExample = () => {
+  return (
+    <Stack orientation="horizontal" spacing="space70">
+        <CornerOrnamentContainer cornerOrnamentType="badge" size="sizeIcon80" position="top_end">
+            <CornerOrnamentBase>
+                <CartIcon color="colorTextIcon" size="sizeIcon80" decorative={false} title="shopping cart" />
+            </CornerOrnamentBase>
+            <CornerOrnament>
+                <ScreenReaderOnly>notification value is</ScreenReaderOnly>
+                <Badge as="span" variant="notification_counter" size="small">
+                    100
+                </Badge>
+            </CornerOrnament>
+        </CornerOrnamentContainer>
+        <CornerOrnamentContainer cornerOrnamentType="icon" size="sizeIcon80">
+            <CornerOrnamentBase>
+                <ProductAlarmsIcon color="colorTextIcon" size="sizeIcon80" decorative={false} title="shopping cart" />
+            </CornerOrnamentBase>
+            <CornerOrnament>
+                <ScreenReaderOnly>You have a notification</ScreenReaderOnly>
+                <NotificationOrnamentIcon color="colorTextIconNotification" decorative size="sizeIcon40" />
+            </CornerOrnament>
+        </CornerOrnamentContainer>
```

Comment:
todo:
- Use NotificationIcon instead of ProductAlarmsIcon
- Set position to top_end